presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -1287,14 +1285,14 @@ protected RelationType visitValues(Values node, AnalysisContext context)
                     Type expectedType = fieldTypes.get(i);
                     Expression item = items.get(i);
                     if (!analysis.getType(item).equals(expectedType)) {
-                        analysis.addCoercion(item, expectedType);
+                        analysis.addCoercion(item, expectedType, false);
```

Comment:
Nice catch. I temporarily changed it to `false` just to make the tests pass, and than i forgot about this place.